src/disk_interface.cc

```diff
@@ -157,13 +158,27 @@ bool DiskInterface::MakeDirs(const string& path) {
 }
 
 // RealDiskInterface -----------------------------------------------------------
+RealDiskInterface::RealDiskInterface() 
+#ifdef _WIN32
+: use_cache_(false), long_paths_enabled_(false) {
+  setlocale(LC_ALL, "");
```

Comment:
How is this related? It causes a regression. See #2435.

Reply:
It seems to be a leftover, thanks for reporting!